techniques/systemSettings/systemManagement/fstabConfiguration/4.0/metadata.xml

```diff
@@ -42,17 +42,18 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
     <SECTION name="Mountpoint" multivalued="true">
       <SELECT1>
         <NAME>FSTAB_ALTERONLY</NAME>
-        <DESCRIPTION>Do you want to add the entry if missing, alter it only or delete it</DESCRIPTION>
+        <DESCRIPTION>Mountpoint state/DESCRIPTION>
+        <LONGDESCRIPTION>Do you want to add the entry if missing, alter it only or delete it</LONGDESCRIPTION>
         <ITEM>
-          <LABEL>Add</LABEL>
+          <LABEL>Present (create if absent)</LABEL>
           <VALUE>false</VALUE>
         </ITEM>
         <ITEM>
-          <LABEL>Alter only</LABEL>
+          <LABEL>Up to date (do not create if absent)</LABEL>
           <VALUE>true</VALUE>
         </ITEM>
         <ITEM>
-          <LABEL>Delete</LABEL>
+          <LABEL>Deleted</LABEL>
```

Comment:
This should be "Absent (delete if present)" in keeping with desired state terminology (and our ncf methods)